MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -107,9 +104,11 @@ -(NSString *)getHostName:(NSString *)urlString {
 # pragma mark AVPlayerItemAccessLog
 
 - (void)handleAVPlayerAccess:(NSNotification *)notif {
-    AVPlayerItemAccessLog *accessLog = [((AVPlayerItem *)notif.object) accessLog];
-    [self handleRenditionChangeInAccessLog:accessLog];
-    [self calculateBandwidthMetricFromAccessLog:accessLog];
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
We should make sure to handle these notifications on the main queue:

>This notification may be posted on a different thread than the one on which the observer was registered.
https://developer.apple.com/documentation/avfoundation/avplayeritemnewaccesslogentrynotification?language=objc